src/chains/ethereum/ethereum/src/blockchain.ts

```diff
@@ -357,10 +357,10 @@ export default class Blockchain extends Emittery<BlockchainTypedEvents> {
 
         //#region automatic mining
         const nullResolved = Promise.resolve(null);
-        const mineAll = (maxTransactions: Capacity, onlyOneBlock = false) =>
+        const mineAll = (maxTransactions: Capacity, onlyOneBlock?: boolean) =>
```

Comment:
I left it as is because we set a default on the `mine` function, and don't feel that requiring it for `mineAll`, but not for `mine` feels nice - we might as well leverage the default that we have specified.